layouts/wrapper.html

```diff
@@ -33,12 +33,20 @@
 		{{/outputBlock}}
 	</head>
 	<body class="o-hoverable-on">
+
+		{{#if showSkipLinks}}
+			<a class="u-visually-hidden" href="#primary-nav">Skip to navigation</a>
+			<a class="u-visually-hidden" href="#content">Skip to content</a>
```

Comment:
I think apps are [currently using](https://github.com/search?q=site-content+user%3AFinancial-Times&type=Code&utf8=%E2%9C%93) the ID `site-content`